### ISSUE TITLE
NHM2 dead after update

### ISSUE THREAD

**Issue by Symbelmyne** [nicehash/NiceHashMiner2#155]
After todays morning update to latest 2.0.1.0 client takes long time on extravator server step (more that 10 mins) and when UI opens its not possible to run any benchmarks cause there are missing files under GPU name. Looks like on installation client no longer download server files, just opens empty.

**Comment by weq** [comment]
I also have this issue.

**Comment by ChilliWinter** [comment]
Also same here
Loading V2.0.1.0 Beta
Starting Excavator Server ...

and nothing

**Comment by ChilliWinter** [comment]
it finally load, but tell me that it is not compatible with AMD card anymore, ok I have to download legacy, but even with the Nvidia it does not load any miner, cause on benchmark it is all empty.
So all broken ?

**Comment by weq** [comment]
Looks like an issue that was in 2.0.7.0 that was resolved back then but it is back I guess.
Bad merge ?

**Comment by ChilliWinter** [comment]
2.0.0.12 was fine, 2.0.1.0 not.
running now 1.8.1.1 legacy awaiting a 2.0.1.0a :)
Thanks

**Comment by dinicthis** [comment]
Yeah, it endlessly cycles between two screens: You hit start and it benchmarks, then when it is finished benchmarking YET AGAIN, it goes back to the same screen where you can hit start.

To get a feel of what it's like, open both these pictures on your computer, and then use alt-tab to cycle between them every twenty seconds, forever.

I think this software is somewhere FAR below beta. Iota?

2.xx.12 works fine, once you get your AV to quit swooping on it with fangs bared.
Same with 1.7.x.53 or whatever. 1.8 blows.


![same screen over and over](https://user-images.githubusercontent.com/30636661/29750677-e0c3038c-8af8-11e7-80b2-f1aa192a3b2b.JPG)
![optimizing yet again](https://user-images.githubusercontent.com/30636661/29750678-e77e2d00-8af8-11e7-8626-ada762d082f7.JPG)

**Comment by Pititul** [comment]
I have the same problem. Tried everything. It even mined for a but and stop.

**Comment by Symbelmyne** [comment]
Today morning there was released an update to 2.0.1.1 so looks like client is fixed. In a fact, it works, there are all benchmarks available including dual mining, too bad there is skunkhash missing.

**Comment by Pititul** [comment]
I still get this issue on 2.0.1.1. :(

**Comment by Symbelmyne** [comment]
Can you try deleting old benchmark file though NHM GUI/configure/logs? After restart, client should generate new benchmark file.

**Comment by Pititul** [comment]
I actually deleted NH folder in AppData/Roaming which deleted all the miners and benchmark files. Doesn;t work.I have been running benchmark all day now today.

**Comment by Symbelmyne** [comment]
What I did - when 2.0.1.0 didnt work I uninstalled it and installed back 2.0.012, today morning while mining I clicked "update your client". Try that.

**Comment by Pititul** [comment]
Thx, I'll try that.

**Comment by dinicthis** [comment]
I installed 2.0.1.1 successfully on both devices with Bitdefender turned OFF. I will see if it survives bitdefender. It should because it uses same directories. No Claymore?

I Installed 2.0.1.1 OVER 2.0.0.12. I did not go from 2.0.1.0.

**Comment by dinicthis** [comment]
I had to specifically allow the executable file in my AV program.

**Comment by dinicthis** [comment]
Further, if you want the fastest miner, EWBF, you have to use NiceHash legacy version 1.8.  It is 10% faster than the others. I was hitting 350 solutions per second mining Zcash, but more like 319 with the other one, was it CCminer? Anyway, EWBF is smokin' fast.

**Comment by Pititul** [comment]
NH 2 doesn't start up for me. Can't fix it. I gonna stick to NH 1.8 for now too. Not much different.
